test/plotter_test.rb

```diff
@@ -3,8 +3,8 @@
 class PlotterTest < Test::Unit::TestCase
   def setup
     @plotter = Charty::Plotter.new(:pyplot)
-    assert_instance_of(Charty::PyPlot,
-                       @plotter.instance_variable_get(:@plotter_adapter))
+    assert_instance_of(Charty::Backends::Pyplot,
+                       @plotter.instance_variable_get(:@backend))
```

Comment:
I think that we can remove this assertion. But this is out of scope for this pull request.